lib/nunjucks/views/manage-prototype/scripts.njk

```diff
@@ -1,3 +1,8 @@
-<script src="/manage-prototype/dependencies/govuk-frontend/govuk/all.js"></script>
-<script src="/manage-prototype/dependencies/govuk-frontend/govuk-prototype-kit/init.js"></script>
-<script src="/plugin-assets/govuk-prototype-kit/lib/assets/javascripts/kit.js"></script>
+{% for scriptConfig in managePlugins.scripts %}
```

Comment:
This seems to be a duplicate of `govuk-prototype-kit/includes/scripts.njk`, and both are taking their input from the plugin config.  We should reuse the existing include rather than copying it.  This can be done by calling the field in the model `pluginConfig` rather than `managePlugins`.